sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/tests/ScenarioTests/BackupVaultTests.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.DataProtectionBackup.Models;
+using Azure.ResourceManager.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.DataProtectionBackup.Tests
+{
+    public class BackupVaultTests : DataProtectionBackupManagementTestBase
+    {
+        public BackupVaultTests(bool isAsync)
+            : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        [Test]
+        [Ignore("Only need to verify serialize and deserialize")]
```

Comment:
We could introduce unit test, just do not inherit the test base class.
And we should not have the `Ignore` attribute
Please move this test class out side the ScenarioTests directory because this is not a scenario test.
Here is an example: https://github.com/Azure/azure-sdk-for-net/blob/605125066d507c13a2491c295535ad4cad364d79/sdk/compute/Azure.ResourceManager.Compute/tests/Unit/GalleryDiskImageTests.cs#L9

Reply:
Fixed